feat(blog): add bpmn-js v16 blog post (#178)
bpmn-io/bpmn.io: src/blog/2023/008-bpmn-js-16-0-0.md

---

> title: Better Labels

Reviewer: ```suggestion
title: Consistent, Better Labels
```

> description: "The bpmn-js@16 and dmn-js@15 releases ships updated translations."

Reviewer: ```suggestion
description: "The bpmn-js@16 and dmn-js@15 releases ship with improved labels. This may impact your custom translations."
```

preview_image: assets/attachments/blog/2023/008-translations.png
layout: blogpost.hbs

> slug: 2023-bpmn-js-16-0-0-translations

Reviewer: ```suggestion
slug: 2023-better-labels
```

</p>

<!-- continue -->

We have updated bpmn-js and dmn-js to use sentence case for all labels to improve readability.

<div class="figure full-size">
  <a href="https://demo.bpmn.io/s/start">
    <img src="{{ assets }}/attachments/blog/2023/008-translations.png" alt="Labels are now sentence case">
  </a>

  <p class="caption">
    Labels are now sentence case.
  </p>
</div>

Along with bpmn-js we have also updated all translations available as part of [bpmn-js-i18n](https://github.com/bpmn-io/bpmn-js-i18n). If you use bpmn-js-i18n in your project, update to the latest version. If you have created custom translations and want to use bpmn-js@16, you need to update them [accordingly](https://github.com/bpmn-io/bpmn-js-i18n/commit/bfa0696804e753ee97af727ec74c6dad4209cf10).

## More Improvements

As always, every release ships with additional, incremental improvements. If you are interested in all the details, go ahead and check out the [full changelog](https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md#1600) to learn about everything new in bpmn-js v16.

## Wrapping up

Did we miss anything? Did you spot a bug, or would you like to suggest an improvement?

> Reach out to us via [our forums](https://forum.bpmn.io/), [follow us on Mastodon](https://fosstodon.org/@bpmn_io), and file an issue you find in the respective project issue trackers.

Reviewer: Cf. https://github.com/bpmn-io/bpmn.io/pull/181

> Reach out to us via [our forums](https://forum.bpmn.io/), tweet us [@bpmn_io](https://twitter.com/bpmn_io), or file an issue you found in [the bpmn-js issue tracker](https://github.com/bpmn-io/bpmn-js/issues).

Reviewer: ```suggestion
Reach out to us via [our forums](https://forum.bpmn.io/), [follow us on Mastodon](https://fosstodon.org/@bpmn_io), and file an issue you find in the respective project issue trackers.
```

> Get the latest BPMN modeling toolkit pre-packaged or as source code via [npm](https://www.npmjs.com/package/bpmn-js) or [unpkg](https://unpkg.com/bpmn-js/).

Reviewer: ```suggestion
```
